apps/voting/app/src/components/VoteCard/VoteCard.js

```diff
@@ -97,11 +97,18 @@ const VoteCard = ({ vote, onOpen }) => {
           ${textStyle('body1')};
           /* lines per font size per line height */
           /* shorter texts align to the top */
-          height: 84px;
+          height: ${10.5 * GU}px;
           display: -webkit-box;
           -webkit-box-orient: vertical;
           -webkit-line-clamp: 3;
           overflow: hidden;
+
+          // overflow-wrap:anywhere and hyphens:auto are not
+          // supported yet by the latest versions of Webkit / Blink,
+          // which is why word-break:break-word has been added.
+          hyphens: auto;
+          overflow-wrap: anywhere;
+          word-break: break-word;
```

Comment:
Should we put this in `VoteText` instead?

Reply:
Yes maybe we could: `VoteText` is inline but it’s only used as a block, the structure must be coming from a need in the previous version. Changing it :+1:

Reply:
Done in https://github.com/aragon/aragon-apps/pull/1021/commits/e6e78b7fa843f10a1099bbaee3b9fdd4f42f3f8e